### ISSUE TITLE
Exported Flow Type?

### ISSUE THREAD

**Issue by zgordon01** [react-native-linear-gradient/react-native-linear-gradient#414]
Is there the flow equivalent of the following typescript interface in `index.d.ts`? If not, is this something we want?

``` typescript
interface LinearGradientProps extends ReactNative.ViewProperties {
        colors: string[],
        start?: { x: number, y: number },
        end?: { x: number, y: number },
        locations?: number[]
    }
```

Thanks in advance.

**Comment by shaundon** [comment]
The Flow types are exported [in this file](https://github.com/react-native-community/react-native-linear-gradient/blob/master/common.js). 

Your code should be able to infer it when you import `LinearGradient`, let me know if it doesn't.

**Comment by zgordon01** [comment]
Yes we have the inferred props on the `LinearGradient` component, but I'm not seeing any type where we can do, for example,

```
let propsForLinearGradient: LinearGradientProps = { ... }
```

Maybe I'm missing something obvious.
